package.nls.json

```diff
@@ -1,4 +1,5 @@
 {
+  "command.activateIcons.title": "Activate VSCode Icon Theme",
```

Comment:
I'm not quite sure this is a correct text. It suggest to activate the theme, but which theme? I mean, if you have 2 or 3 themes this doesn't identify our theme. 

Our theme is called `vscode-icons`, so I guess something like `Activate vscode-icons Icon Theme` would be a more suitable.

What do you think?